github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -194,6 +194,49 @@ periodics:
     - name: gcs
       secret:
         secretName: gcs
+- name: periodic-kubevirt-e2e-k8s-1.19-sysprep
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+    testgrid-alert-email: bodnopoz@redhat.com, kubevirtci-sysprep-test@googlegroups.com
+    testgrid-num-failures-to-alert: "1"
+    testgrid-alert-stale-results-hours: "1"
+  cron: "0 0 * * *"
+  decorate: true
+  decoration_config:
+    timeout: 4h
+    grace_period: 5m
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+  extra_refs:
```

Comment:
This is not required, being created in this dir by default the workdir will be kubevirt/kubevirt master.

Reply:
fixed